src/modules/createEvent/CreateEvent.tsx

```diff
@@ -86,8 +86,27 @@ const CreateEvent = ({ navigation, route }: CreateEventProps) => {
 
   const schema = data['eventSchemas'][route.params.type.toLowerCase()];
 
+  function hasAllEventDetails(eventData: any) {
+    return (
```

Comment:
I don't think this works quite right since different event types have different fields. For example, Special Events don't have a distance field, so with this logic it's impossible to create one. There's probably some way to check all of the fields specific to the event type?